### PR TITLE
[TransformText] Added support for transliterating text into other scripts

### PR DESCRIPTION
**Pull request by pvieito**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add the rest too?

- latinToHangul
- latinToHiragana
- latinToKatakana
- latinToThai
- hiraganaToKatakana
- mandarinToLatin

I think `stripDiacritics` could also be useful in some cases.

File: IntentsExtension/Intents.intentdefinition [hunk 1]
```diff
@@ -75,6 +75,56 @@
 					<key>INEnumValueName</key>
 					<string>dashCase</string>
 				</dict>
+				<dict>
+					<key>INEnumValueDisplayName</key>
+					<string>Transliterate to Arabic Script</string>
```
Comment (sindresorhus) [inline IntentsExtension/Intents.intentdefinition:80]: You are assuming the input is always Latin.

```suggestion
					<string>Transliterate Latin script to Arabic script</string>
```

Comment (sindresorhus) [reply IntentsExtension/Intents.intentdefinition:80]: Sorry, I don't understand your reply. Can you elaborate?

**Comment by pvieito** [comment]
@sindresorhus Done!

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**